examples/check-if-testcafe-active/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Check for TestCafe</title>
+</head>
+<body>
```

Comment:
```js
<body>
     <div>
            The page is opened with TestCafe: <span id="result"></span>
     </div>
     <script>
            const resultSpan   = document.getElementById('result');
            const isInTestCafe = !!window['testCafeDriverInstance%'];
            
            resultSpan.textContent = isInTestCafe ? 'yes' : 'no';
     </script>
</body>
```

Reply:
Doesn't quite work- shows `no`, with testcafe or without. This seems to work though:

> const isInTestCafe = !!window.localStorage.nativeStorageKey.startsWith('hammerhead');